client/src/components/related/RelatedProductsModule.jsx

```diff
@@ -1,10 +1,34 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import RelatedProductList from './lists/RelatedProductList.jsx';
+import YourOutfitList from './lists/YourOutfitList.jsx';
+import api from '../../../../client/src/lib/api.js';
 
 const RelatedProductsModule = () => {
+  const [relatedItems, setRelatedItems] = useState(['no related products']);
+  const [outfitList, setOutfitList] = useState([]);
+
+  const getProductList = (err, result) => {
+    api.listProducts()
+    .then((res) => {
+      setRelatedItems(res);
+      console.log(res);
+    })
+    .catch((err) => {
+      console.log('get product failed', err);
+    })
+  }
+
+  useEffect(getProductList, []);
 
   return (
     <div>
```

Comment:
Not urgent, but let's update these divs to use semantic HTML where applicable. I think an easy one would be converting the outermost div to a section, but let me know if you disagree!

Reply:
I can note it and do some sematic updates for my next pull request.